doc/12-icinga2-api.md

```diff
@@ -2028,9 +2109,9 @@ by sending a `GET` request to the URL endpoint `/v1/config/files`. You must incl
 the package name, stage name and the `startup.log` in the URL path.
 
 ```
-$ curl -k -s -u root:icinga 'https://localhost:5665/v1/config/files/example-cmdb/7e7861c8-8008-4e8d-9910-2a0bb26921bd/startup.log'
-...
-
+$ curl -k -s -S -i -u root:icinga 'https://localhost:5665/v1/config/files/example-cmdb/7e7861c8-8008-4e8d-9910-2a0bb26921bd/startup.log'
+```
+```
```

Comment:
I don't know what happens without a newline in here. Better include one for consistency.

I presume the `...` showed that the response body was already truncated (didn't explicitly verify it myself but startup.log should usually contain more than a single line), so I'd keep it within the new block, or even better, make it a `[...]` as this is more common to show omissions IMHO.